sdbuild/Makefile

```diff
@@ -210,7 +210,7 @@ $$(BASE_$1) : $$(STAGE2_$1) $$(STAGE3_DEPENDS_$1) | $$(IMAGE_ROOT)
 	$$(SCRIPT_DIR)/unmount_image.sh $$(STAGING_$1) $$(BASE_$1)
 
 $$(STAGE1_$1): $$(CONFIG_$1)/multistrap.config | $$(BUILD_ROOT) $$(CCACHEDIR)
-	-rm $$@
+	-rm -rf $$@
```

Comment:
I think this should just be -f as we don't want to start accidentally deleting things recursively.